superset-frontend/src/features/alerts/components/AlertReportCronScheduler.tsx

```diff
@@ -66,9 +70,45 @@ export const AlertReportCronScheduler: React.FC<AlertReportCronSchedulerProps> =
 
     return (
       <>
-        <Radio.Group onChange={handleRadioButtonChange} value={scheduleFormat}>
-          <div className="inline-container add-margin">
-            <Radio data-test="picker" value="picker" />
+        <StyledInputContainer>
+          <div className="control-label">
+            {t('Schedule type')}
+            <span className="required">*</span>
+          </div>
+          <div className="input-container">
+            <Select
+              ariaLabel={t('Schedule type')}
+              placeholder={t('Schedule type')}
+              onChange={(e: any) => {
```

Comment:
Can we replace any with `'picker' | 'input'`? (that could we moved to a separate type to avoid repetition with line 46)

Reply:
Actually it could be an enum - we're using those 2 strings in `SCHEDULE_TYPE_OPTIONS`, in `scheduleFormat` and then in a conditional in line 98